install-steps.md

```diff
@@ -618,6 +618,145 @@ podman run -d --name rhcos_image_cache \
 registry.centos.org/centos/httpd-24-centos7:latest
 ~~~
 
+## Create a Disconnected Registry (optional)
+At times, it may be desirable to install an Openshift KNI cluster using a local copy of the installation registry.
+This could be for network efficiency or the cluster to be deployed is on a network that does not have access to the internet.
+
+A local, or mirrored, copy of the registry requires the following:
+* A certificate for the registry host, this can be self-signed.
+* A webserver, this will be served by a container on a system.
+* An updated pull secret that contains the certificate and local repository information.
+
+### Prepare the System to Host the Mirrored Registry
+Open the firewall for the registry.
+```bash
+$ sudo firewall-cmd --add-port=5000/tcp --zone=libvirt  --permanent
+$ sudo firewall-cmd --add-port=5000/tcp --zone=public   --permanent
+$ sudo firewall-cmd --add-service=http  --permanent
```

Comment:
line 635 already captured in the original firewall rules.

Reply:
removed. will be in next commit.

Reply:
commit pushed